packages/changeset-bot/index.js

```diff
@@ -29,7 +29,9 @@ module.exports = app => {
 
     Promise.all([getFiles, getLatestCommit]).then(
       ([hasChangeset, latestCommit]) => {
-        const user = latestCommit.committer.login;
+        const user = latestCommit.committer
+          ? latestCommit.committer.login
+          : latestCommit.commit.author.name;
```

Comment:
Just checking, is this always present? Should we ever worry about its absence?

Reply:
I can't imagine it not always being present and I didn't want to be unnecessarily defensive if I could avoid it, but if you don't think the same then I'm happy to change it to keep the empty case.

Reply:
Nah, I'm happy to trust this. Next sprint or a couple of sprints time, we should talk about making the logging slightly better so it's easier for us to debug problems in the future.